docs/en/deploy-loom-testnet.md

```diff
@@ -10,13 +10,13 @@ This doc is for developers, and it aims to show how they can join the Loom Testn
 
 ## Overview
 
-Deploying to Loom PlasmaChain is no different from deploying to the Ethereum main net, or to Rinkeby. If you know how to do one, you also know how to do the other.
+Deploying to Loom Basechain is no different from deploying to the Ethereum main net, or to Rinkeby. If you know how to do one, you also know how to do the other.
```

Comment:
Deploying to Loom Basechain --> Deploying to Basechain
the Ethereum main net --> Ethereum mainnet